src/team/g6/DemoG6.java

```diff
@@ -63,6 +63,15 @@ public static void addition(int c,int d) { //REPOSITORY
 	public static void goodBye() { //LOCAL
 		String goodBye;
 			// just another test
+	
+		System.out.println("New Edits for Review");
+		
+		System.out.println("It's a mad world");
 		
+		System.out.println("beep boop beep boop");
 	}
+	
+	
+	// Development Stage 123
```

Comment:
Nice addition - please go over the conflicts/comment, and see what you'd like to do.